go/host/rpc/clientapi/client_api_eth.go

```diff
@@ -49,11 +49,24 @@ func (api *EthereumAPI) GetBalance(_ context.Context, encryptedParams common.Enc
 
 // GetBlockByNumber returns the rollup with the given height as a block. No transactions are included.
 func (api *EthereumAPI) GetBlockByNumber(_ context.Context, number rpc.BlockNumber, _ bool) (map[string]interface{}, error) {
+	var rollupHeaderWithHashes *common.HeaderWithTxHashes
+
+	switch number {
+	case rpc.LatestBlockNumber:
+		return headerWithHashesToBlock(api.host.DB().GetCurrentRollupHead()), nil
+	case rpc.EarliestBlockNumber:
+		rollupHeaderWithHashes = api.host.DB().GetRollupHeader(gethcommon.BigToHash(big.NewInt(0)))
```

Comment:
Just double-checking - we def want the genesis rollup, not the first produced rollup, right?